sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -471,6 +471,24 @@ def test_group_by_key(self):
     assert_that(result, equal_to([(1, [1, 2, 3]), (2, [1, 2]), (3, [1])]))
     pipeline.run()
 
+  def test_group_by_key_reiteration(self):
+    class MyDoFn(beam.DoFn):
+      def process(self, gbk_result):
+        value_list = gbk_result[1]
```

Comment:
More idiomatic to do `key, value_list = gbk_result`

Reply:
Thanks! Fixed.